Makefile

```diff
@@ -0,0 +1,9 @@
+test: build
```

Comment:
seems like there's a bit of a forge bug where the vault tests hang (maybe a result of them all inheriting the same abstract contract ?), but this runs each of them sequentially